ci/releases.yml

```diff
@@ -32,6 +32,25 @@ suite codesign deploy staging-suite:
   tags:
     - deploy
 
+# Suite desktop deploy to staging
+suite-desktop deploy autoupdate test:
+  stage: deploy to dev
+  needs:
+    - suite-desktop build mac codesign
+    - suite-desktop build linux codesign
+    - suite-desktop build windows codesign
+  only:
+    refs:
+      - codesign
+  variables:
+    DEPLOY_DIRECTORY: ${DEPLOY_BASE_DIR}/suite-desktop/${CI_BUILD_REF_NAME}
+  script:
+    - mkdir -p ${DEPLOY_DIRECTORY}
+    - 'rsync --delete -va Trezor-Suite* "${DEPLOY_DIRECTORY}/" || :'
```

Comment:
Would be really nice be able to have more than one build available at the same time... What about removing the `delete` flag? 🤔

Reply:
what would be the benefit of that? we can still only update to the highest version no?

Reply:
aha, never mind, I didn't realize, that we still have other place where we could download codesign builds from (artifacts)